BrowserStack.java

```diff
@@ -37,32 +38,113 @@ public static void myassert(boolean  x) {
     //push() - pushes the element in this case its a url (String)
     private  void push(String url){
         //WRITE YOUR CODE HERE
+    	
```

Comment:
However, I am not sure if ArrayList in Java will dynamically increase the capacity even beyond initial size. Question for Java experts.